src/models/Campaign.js

```diff
@@ -56,6 +57,10 @@ class Campaign extends BasicModel {
     return campaign;
   }
 
+  get type() {
+    return this._type;
+  }
+
```

Comment:
there is already a static getter for type. Does that not work for instances? If not, we should at least delegate the call to `Campaign.type()` instead of adding `_type` to the instance